registry/storage/blobwriter.go

```diff
@@ -89,11 +91,12 @@ func (bw *blobWriter) Cancel(ctx context.Context) error {
 		return err
 	}
 
+	bw.Close()
```

Comment:
Should this error still be ignored or is `removeResources` after commit already harmless?